sensors.py

```diff
@@ -20,28 +25,93 @@ def get_reading(self, sensor_type: SensorType) -> int | float:
         raise NotImplementedError
 
 class DHT(Sensor):
+    def __init__(self) -> None:
+        super().__init__()
+        self.dht = DHT22(board.D4)
+
     def get_reading(self, sensor_type: SensorType) -> int:
         with self._lock:
-            if sensor_type == SensorType.TEMPERATURE:
-                return randint(10, 30)
-            if sensor_type == SensorType.HUMIDITY:
-                return randint(40, 100)
+            try:
+                if sensor_type == SensorType.TEMPERATURE:
+                    return self.dht.temperature
+                if sensor_type == SensorType.HUMIDITY:
+                    return self.dht.humidity
+            except:
+                logging.exception("Couldn't get a reading")
+                raise SensorReadingError
         raise WrongSensorType
```

Comment:
Is that necessary?

Reply:
dht.humidity/temperature has return type int | float | None.
In current version(you commented outdated one), I raise SensorReadingError if somehow dht returned None